docs/reference/extensibility/NuGet-Credential-Providers-for-Visual-Studio.md

```diff
@@ -14,6 +14,8 @@ After you install a NuGet credential provider for Visual Studio, the NuGet Visua
 
 A sample implementation can be found in [the VsCredentialProvider sample](https://github.com/NuGet/Samples/tree/master/VsCredentialProvider).
 
+The NuGet Visual Studio Extension uses an internal `VsCredentialProviderImporter` which also scans for plug-in credential providers. These plug-in credential providers must be discoverable as a MEF Export of type `IVsCredentialProvider`.
```

Comment:
As mentioned in an earlier comment, I think calling NuGet an extension might be confusing.

```suggestion
Within Visual Studio, NuGet uses an internal `VsCredentialProviderImporter` which also scans for plug-in credential providers. These plug-in credential providers must be discoverable as a MEF Export of type `IVsCredentialProvider`.
```